be/src/vec/sink/vdata_stream_sender.h

```diff
@@ -313,12 +182,54 @@ class PipChannel final : public Channel<pipeline::ExchangeSinkLocalState> {
 
     std::shared_ptr<pipeline::Dependency> get_local_channel_dependency();
 
-private:
+protected:
+    Status _send_local_block(bool eos);
+    Status _send_current_block(bool eos);
+
+    bool _recvr_is_valid() {
```

Comment:
should we return Status in this func and dispose the `_receiver_status` out the function is better ?